problems/sevenBoom.js

```diff
@@ -11,8 +11,17 @@
 * 17 is also replaced with 'BOOM' because it contains a 7.
 */
 
-function sevenBoom() {
-
+function sevenBoom(n) {
+    let newArr = []
+    for (let i = 1; i <= n; i += 1) {
+        if (i % 7 === 0 || i % 10 === 7) {
```

Comment:
Consider numbers 70-79; will they be 'BOOM' based on this function? How can you find any number that contains a 7 in any digit?